test/nuxeo-user-suggestion.html

```diff
@@ -0,0 +1,103 @@
+<!--
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html>
+<head>
+  <title>nuxeo-directory-widgets</title>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+  <script>
+    // force use of polyfill: see ELEMENTS-56
+    window.fetch = null;
+  </script>
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+  <script src="../../nuxeo-elements/test/test-helpers.js"></script>
+
+  <link rel="import" href="../../polymer/polymer.html">
+  <link rel="import" href="../../iron-test-helpers/mock-interactions.html">
+  <link rel="import" href="../../nuxeo-elements/nuxeo-connection.html">
+  <link rel="import" href="../../nuxeo-elements/nuxeo-operation.html">
+  <link rel="import" href="../widgets/nuxeo-user-suggestion.html">
+</head>
+<body>
+  <test-fixture id="nx">
+    <template>
+      <nuxeo-connection url="/dummy"></nuxeo-connection>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="prefixed">
+    <template>
+        <nuxeo-user-suggestion multiple prefixed>
+        </nuxeo-user-suggestion>
+    </template>
+  </test-fixture>
+
+  <script>
+    suite('<nuxeo-user-suggestion>', () => {
+
+      test('prefixed mode intialization', () => {
```

Comment:
Nice to have tests! 👍 
Should we also include one test with "prefixed_id"?

Reply:
`prefixed_id` will never be used at init when we bind the field to a resolved field. It is only used when you type something in the input and the widget is fed by the operation.
But you somehow raise the point that tests were missing on this widget. Here I am adding a non-regression test. Maybe I can create dedicated ticket to add test coverage on this widget.